build_projects/dotnet-cli-build/SetEnvVar.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class SetEnvVar : Task
```

Comment:
We need this because this is the only way to set "Telemetry" options (i.e. the telemetry profile).  We have an issue to be able to set these things using a config file:  https://github.com/dotnet/cli/issues/2960.